src/content/structured/get-started/custom-theme.mdx

```diff
@@ -39,9 +44,144 @@ Alternatively, you can include the `ic-theme` component and set the `color` prop
+    padding: "var(--ic-space-sm)"
+  }
+</style>
+<body>
+  <div>
+    {shortCode}
+  </div>
+  <script>
+    function handleClick() {
+      theme = document.querySelector("ic-theme");
+      theme.theme = theme.theme === "light" ? "dark" : "light";
+    }
+  </script>`,
+    },
+  },
+  {
+    technology: "React",
+    snippets: {
+      short: `<IcButton onClick={handleClick}>Update ic-theme</IcButton>
+<IcTheme theme={theme} className={classes.themeContainer}>
+  <IcAlert heading="Set at component level" variant="info" theme="light" />
+  <IcAlert heading="Inherits from ic-theme" variant="info" />
+</IcTheme>
+<IcAlert heading="Inherits from system/browser settings" variant="info" />`,
+      long: [
+        {
+          language: "Typescript",
+          snippet: `const [theme, setTheme] = useState<"light" | "dark">("light");
+const handleClick = () => {
+  setTheme(theme === "light" ? "dark" : "light");
+};
+const useStyles = createUseStyles({
+  themeContainer: {
+    border: "1px solid var(--ic-architectural-white)",
+    padding: "var(--ic-space-sm)"
+  },
+});
+const classes = useStyles();
+<>
+  {shortCode}
+</>`,
+        },
+        {
+          language: "Javascript",
+          snippet: `const [theme, setTheme] = useState("light");
+const handleClick = () => {
+  setTheme(theme === "light" ? "dark" : "light");
+};
+const useStyles = createUseStyles({
+  themeContainer: {
+    border: "1px solid var(--ic-architectural-white)",
+    padding: "var(--ic-space-sm)"
+  },
+});
+const classes = useStyles();
+<>
+  {shortCode}
+</>`,
+        },
+      ],
+    },
+  },
+];
+
+export const DarkModeExample = () => {
+  const [theme, setTheme] = useState("light");
+  const handleClick = () => {
+    setTheme(theme === "light" ? "dark" : "light");
+  };
+  return (
+    <>
+      <IcButton onClick={handleClick}>Update ic-theme</IcButton>
+      <IcTheme
+        theme={theme}
+        style={{
+          border: "1px solid var(--ic-architectural-white)",
+          padding: "var(--ic-space-sm)",
+        }}
+      >
+        <IcAlert
+          heading="Set at component level"
+          variant="info"
+          theme="light"
+        />
+        <IcAlert heading="Inherits from ic-theme" variant="info" />
+      </IcTheme>
+      <IcAlert heading="Inherits from system/browser settings" variant="info" />
+    </>
+  );
+};
+
+<ComponentPreview
+  style={{
+    flexDirection: "column",
+    gap: "var(--ic-space-sm)",
+    background: "#333333",
+  }}
+  snippets={snippetsDefault}
+>
+  <DarkModeExample />
+</ComponentPreview>
+
+## Monochrome
+
+Another part of ICDS version 3.0, the `appearance` prop has been replaced with a boolean `monochrome` prop. This prop performs a similar function, to remove the colour from components to help them display on certain backgrounds, however the key difference is that by setting `monochrome` to `true`, it will display as black and white whilst adjusting to the dark mode inheritance mentioned above.
```

Comment:
```suggestion
This prop removes the colour from components to help them display on certain backgrounds, by setting `monochrome` to `true` it will display as black and white whilst adjusting to the dark mode inheritance mentioned above.
```